docs/index.rst

```diff
@@ -1580,6 +1660,249 @@ Creates a new release for dialogue ``dialogue_id`` under the project with the id
       "url": "/projects/23/dialogues/21/releases/1"
     }
 
+.. _channels:
+
+Channels and Providers
+----------------------
+
+.. http:get:: /channels/
+
+  Retrieves the :ref:`descriptions <data-channels>` of all channels. Only
+  accessible if the authenticated user has :ref:`admin permission
+  <permissions-admin>`.
+
+  .. sourcecode:: http
+
+     GET /channels/ HTTP/1.1
+
+  .. sourcecode:: http
+
+     HTTP/1.1 200 OK
+     Content-Type: application/json
+
+     [{
+       id: '23',
+       url: '/channels/23',
+       project_id: 21,
+       title: '@foo',
+       address: '@foo',
+       type: 'twitter',
+       is_available: true,
+       provider: {
+         id: '21',
+         title: 'Twitter'
+       }
+     }]
+
+.. http:get:: /channels/(str:channel_id)
+
+  Retrieves the :ref:`description <data-channels>` of the channel with the id
+  ``channel_id``.
+
+  Only accessible if the authenticated user has :ref:`admin permission
+  <permissions-admin>` or has access to a project using the channel
+  ``channel_id``.
+
+  .. sourcecode:: http
+
+     GET /channels/21 HTTP/1.1
+
+  .. sourcecode:: http
+
+     HTTP/1.1 200 OK
+     Content-Type: application/json
+
+     {
+       id: '23',
+       url: '/channels/23',
```

Comment:
Should this be `/channels/21` since that is the channel being retrieved by the example?